host/4/publish-appservice-stage-sovereign-clouds.yml

```diff
@@ -147,16 +147,19 @@ steps:
           docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.7-appservice
           docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.8-appservice
           docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.9-appservice
+          docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.10-appservice
           
           docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.7-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.7-appservice-$(CloudName)-stage${{stage}}
           docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.8-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.8-appservice-$(CloudName)-stage${{stage}}
-          docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.9-appservice $TARGET_REGISTRY/python:$(TargetVersion)-appservice-$(CloudName)-stage${{stage}}
           docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.9-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.9-appservice-$(CloudName)-stage${{stage}}
+          docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.10-appservice $TARGET_REGISTRY/python:$(TargetVersion)-appservice-$(CloudName)-stage${{stage}}

```

Comment:
@balag0 brought up a good point - let's leave base images as 3.9 until 3.10 is GA and is made default